js/chessMoveReminder.js

```diff
@@ -94,7 +91,22 @@ function chessMoveReminder() {
                             } else {
                                 timeToWait = number * 1000;
                             }
-                            timer = setTimeout(playAudio, timeToWait, who);
+                            timer = setTimeout(playAudio, timeToWait, formattedWho);
+                            
+                            if (repeatEnabled) {
+                                setTimeout(function(){
+                                    if (repeatType === 'percentage') {
+                                        let seconds =
+                                        parseSecondsFromClock(currentClock) *
+                                        repeatNumber *
+                                        10;
+                                        timeToWait = Math.max(seconds, 2000);
+                                    } else {
+                                        timeToWait = repeatNumber * 1000;
```

Comment:
Try out 0 second repeat :D

We should probably set a floor for this repeat (what's the longest sound clip that we have?)